packages/webapp/pages/squads/[handle]/moderate.tsx

```diff
@@ -0,0 +1,87 @@
+import React, { ReactElement, useEffect } from 'react';
+import {
+  ManageSquadPageContainer,
+  SquadSettingsProps,
+} from '@dailydotdev/shared/src/components/squads/utils';
+import {
+  SquadTab,
+  SquadTabs,
+} from '@dailydotdev/shared/src/components/squads/SquadTabs';
+import { SquadModerationList } from '@dailydotdev/shared/src/components/squads/moderation/SquadModerationList';
+import {
+  PageHeader,
+  PageHeaderTitle,
+} from '@dailydotdev/shared/src/components/layout/common';
+import {
+  Button,
+  ButtonVariant,
+} from '@dailydotdev/shared/src/components/buttons/Button';
+import { ArrowIcon } from '@dailydotdev/shared/src/components/icons';
+import {
+  GetStaticPathsResult,
+  GetStaticPropsContext,
+  GetStaticPropsResult,
+} from 'next';
+import { ParsedUrlQuery } from 'querystring';
+import { useSquad } from '@dailydotdev/shared/src/hooks';
+import { useRouter } from 'next/router';
+import { getLayout as getMainLayout } from '../../../components/layouts/MainLayout';
+
+export default function ModerateSquadPage({
+  handle,
+}: SquadSettingsProps): ReactElement {
+  const { squad, isFetched } = useSquad({ handle });
+  const router = useRouter();
+
+  useEffect(() => {
+    if (!isFetched) {
+      return;
+    }
+
+    if (squad?.moderationRequired) {
+      return;
+    }
+
+    router.push(squad ? squad.permalink : '/404');
+  }, [squad, isFetched, router]);
+
+  return (
+    <ManageSquadPageContainer>
+      <PageHeader className="border-b-0">
```

Comment:
I see in many places we need to overwrite to 0 border bottom, wouldn't it make more sense to not have a border and then pass the one place you need it?

Reply:
This is used somewhere where the `border-b` is necessary. Also, that is the default behavior for headers, it just so happened here we reused it on top another header (tabs).

Reply:
Ah ok, lets leave it as is then yes.